README.md

```diff
@@ -36,25 +36,39 @@ output at the same time it applies them for applications output:
 9. Output Limiter (Ladspa Fast Lookahead Limiter)
 10. Spectrum Analyzer (Gstreamer)
 
-## Installation
+## Build and install
 
 Users upgrading from 1.x to 2.x will have to rebuild their presets. Since
 version 2.0.0 PulseEffects uses a different format. This
 change was necessary to support presets for microphone processing.
 
+```
+$ meson _build --prefix=/usr
+$ cd _build
+$ sudo ninja install
+```
+
 ### GNU/Linux Packages
 
 - [Arch Linux](https://aur.archlinux.org/packages/pulseeffects/)
 
+Ubuntu >=17.10 repository:
+```
+wget -q -O- http://repo.dumalogiya.ru/keys/mikhailnov_pub.gpg | sudo apt-key add -
+echo "deb http://repo.dumalogiya.ru/aptly/public artful main" | sudo tee /etc/apt/sources.list.d/dumalogiya-artful.list
+sudo apt update
+sudo apt install pulseeffects
+```
+
 ### Flatpak
 
 [Flatpak](https://flatpak.org) packages support multiple distributions and are sandboxed.
 
 Stable releases are hosted on [Flathub](https://flathub.org):
 
 ```
-flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
-flatpak install flathub com.github.wwmm.pulseeffects
+$ flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

Comment:
Any reason you added $ here but not on your own additions?

Reply:
I wanted to show that the commands are expected to be ran not under root, but did not follow the same style in all places

Reply:
Interesting is that what the dollar sign means in these instances? I am quite experienced with Linux and Open Source and have seen the $ sign used but never knew why. I assumed it was there to make it harder for people to just copy/paste, since that can be dangerous.

Is there any documentation, or other reading material I have missed, that you can link to on this?

Of course you are right in this case then, since flatpak is best used without root or sudo.

Reply:
$ is a standart command prompt fot non-root users and # is for root in bash and other shells.
Just execute 
``` sudo -i ```
You login prompt will be changed to # from $.

So, if the command starts with #, users understand that the command is expected to execute as root.